can/interfaces/vector/canlib.py

```diff
@@ -363,22 +384,25 @@ def reset(self):
     @staticmethod
     def _detect_available_configs():
         configs = []
-        if vxlapi is None:
-            return configs
-        driver_config = vxlapi.XLdriverConfig()
-        try:
-            vxlapi.xlOpenDriver()
-            vxlapi.xlGetDriverConfig(driver_config)
-            vxlapi.xlCloseDriver()
-        except:
-            pass
-        LOG.info('Found %d channels', driver_config.channelCount)
-        for i in range(driver_config.channelCount):
-            channel_config = driver_config.channel[i]
+        channel_configs = get_channel_configs()
+        LOG.info('Found %d channels', channel_count)
```

Comment:
Change `channel_count` to `len(channel_configs)` (or remove line since it does not provide additional information).